wagtail/admin/templates/wagtailadmin/shared/set_privacy.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n wagtailadmin_tags %}
+
+<form action="{% url action_url_name object.id %}" method="POST" novalidate data-controller="w-rules" data-action="change->w-rules#resolve">
```

Comment:
Sorry I should've mentioned in the original comment, can we do the `reverse()` from the view instead of the template? It allows the template to be more , e.g. if the view URL pattern requires more arguments

```suggestion
<form action="{{ action_url }}" method="POST" novalidate data-controller="w-rules" data-action="change->w-rules#resolve">
```

(need update for the tests too)

Reply:
OK this should be good now, ready for another review also.